releases.json

```diff
@@ -931,6 +931,7 @@
       "glut"
     ],
     "versions": [
+      "3.4.0-4",
```

Comment:
Could be a separate commit?

Reply:
It doesn't seem like that's the norm from the commit history, is it?